test/Common/TestEntityClasses.cs

```diff
@@ -553,10 +567,17 @@ public class UnDeserializableKaboom : Exception
         [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
         public class StorageBackedCounter : IAsyncCounter
         {
+            private const string BlobName = "counter";
+
+#if FUNCTIONS_V1
             private readonly CloudBlobContainer blobContainer;
-            private readonly string blobName = "counter";
 
             public StorageBackedCounter(CloudBlobContainer blobContainer)
+#else
+            private readonly BlobContainerClient blobContainer;
+
+            public StorageBackedCounter(BlobContainerClient blobContainer)
```

Comment:
Do you happen to know if a similar change exists for non-Durable Function apps? Do users in Functions V3+ bind to the newer storage SDK types? It seems like this is a breaking change.

Reply:
Oh no, I think they should be fine. I updated the version of the blob trigger library, and with it the types that could be bound. I wanted to remove as many vestiges of the old SDK as possible haha